workspace.json

```diff
@@ -4087,8 +4087,8 @@
         "migrate": {
           "builder": "@nrwl/workspace:run-commands",
           "options": {
-            "command": "node migrate.js",
-            "cwd": "dist/apps/services/search-indexer"
+            "command": "yarn ts-node -P apps/services/search-indexer/tsconfig.app.json apps/services/search-indexer/src/migrate/index.ts",
```

Comment:
Can skip yarn

Reply:
I need yarn, were you thinking of `package.json`?

Reply:
No, thought you wouldn't need it since you run nx through yarn.. but it was nit-picky, can ignore.